external/kegg/getKEGGModelForOrganism.m

```diff
@@ -595,16 +564,23 @@
 missingAligned=setdiff(KOModel.rxns,[alignedFiles;hmmFiles;alignedWorking;outFiles]);
 if ~isempty(missingAligned)
     if seqIdentity==-1
-        fprintf('Performing the multiple alignment for KEGG Orthology specific protein sets... ');
+        fprintf('Performing the multiple alignment for KEGG Orthology specific protein sets...   0%% complete');
     else
-        fprintf('Performing clustering and multiple alignment for KEGG Orthology specific protein sets... ');
+        fprintf('Performing clustering and multiple alignment for KEGG Orthology specific protein sets...   0%% complete');
     end
     missingAligned=missingAligned(randperm(RandStream.create('mrg32k3a','Seed',cputime()),numel(missingAligned)));
-    progressFlag=0;
-    %Update fastaFiles. This is needed once rebuilding KEGG from FTP dump
-    %files for more accurate progress reporting
-    fastaFiles=listFiles(fullfile(dataDir,'fasta','*.fa'));
-    %Align all sequences using MAFFT
+    tmpFile=tempname;
+    %On Windows, paths need to be translated to Unix before parsing it to WSL
+    if ispc
+        wslPath.tmpFile=getWSLpath(tmpFile);
+        %mafft has problems writing to terminal (/dev/stderr) when running
+        %on WSL via MATLAB, instead write and read progress file
+        mafftOutput = tempname;
+        wslPath.mafftOutput=getWSLpath(mafftOutput));
```

Comment:
```suggestion
        wslPath.mafftOutput=getWSLpath(mafftOutput);
```

Reply:
Fixed in [ce329ba](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/ce329ba245412b7f7f0868c985efb1a39a61a2ea)